core/src/wallet.proto

```diff
@@ -132,6 +132,9 @@ message Key {
 
   // Encrypted version of the seed
   optional EncryptedData encrypted_deterministic_seed = 9;
+
+  // The path to the root.
+  repeated uint32 originalAccountPath = 10 [packed=true];
```

Comment:
For consistency, I think this should be named "original_account_path". What does the "original" mean here, isn't it just an "account_path"? Should it perhaps be optional, because existing wallets don't have this?

Reply:
I actually changed the name of the field because it was not clear (i originally had the name used previously).
https://github.com/uniquid/bitcoinj/pull/4

What we want is the cointype path (44H 1H 0H), which is what we save. 
https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#Path_levels

Reply:
From the perspective of bitcoinj and BIP32, there is no cointype. "account path" is the more correct term, and consistent with what we use in bitcoinj already. I was only wondering about the "original" prefix.

Reply:
...and even if I look at the spec you're quoting, the level the current field points at seems to be "account" rather than "cointype".

Reply:
Just saw that i misread. It is indeed the accountPath that we want and have. original is not needed.

I will update my commits to reflect that.